### ISSUE TITLE
Wildcards in SDID of a rule?

### ISSUE THREAD

**Issue by mjdl** [lieser/dkim_verifier#77]
Many organizations out-source email blasts to other companies. These in turn usually have multiple servers DKIM signing emails, e.g. the blast email with From `user@example.com` may be signed by any of `emailblast01.com` `emailblast02.com`, etc., each with their own certificate.

Is it possible to have wildcards in the SDID of a user-specified rule? E.g. `emailnblast*.com` in the example above?

**Comment by lieser** [comment]
No, at the moment this is not possible. And even If this would be possible, I would not advise to use it, as this could lead to domains being included than are not under the control of the intended organizations.

If you still think this would be useful for you, I could maybe add an advanced option for this.

**Comment by mjdl** [comment]
I understand the security concerns, and probably it's not such a good enhancement, especially since a wildcard `*` character can concatenate any string. But when the domain name of the `From` and the actual `SDID` are completely different (rule can't be automatically added), which occurs quite often, maybe a truly geeky option of RegEx might be fun...
